src/test/java/teammates/test/cases/search/StudentSearchTest.java

```diff
@@ -88,14 +88,14 @@ public void allTests() {
         ______TS("success: search for students; deleted student without deleted document: the document "
```

Comment:
You should modify/remove the test case as fit

Reply:
I removed the test case since deletion is already checked